cluster/gce/gci/configure-helper-common.sh

```diff
@@ -1778,7 +1778,10 @@ function start-kube-apiserver {
   fi
   echo "APISERVER_SERVICEGROUPID: ${APISERVER_SERVICEGROUPID:-0}"
   params+=" --service-group-id=${APISERVER_SERVICEGROUPID:-0}"
-  params+=" --partition-config=${KUBE_HOME}/apiserver.config"
+  if [[ -f "${KUBE_HOME}/apiserver.config" ]]; then
+    mv ${KUBE_HOME}/apiserver.config /etc/srv/kubernetes/
+    params+=" --partition-config=/etc/srv/kubernetes/apiserver.config"
```

Comment:
Can /etc/srv/kubernetes/ be a constant? It seems used a lot. It does not need to be done now. It can be done later for code refactor.